File: app/src/main/java/com/piyushsatija/pollutionmonitor/ApiInterface.java
package com.piyushsatija.pollutionmonitor;

import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Path;
import retrofit2.http.Query;

public interface ApiInterface {

    @GET("feed/here/")
    Call<ApiResponse> getAQI(@Query("token") String token);

    @GET("feed/{geo}/")
    Call<ApiResponse> getLocationAQI(@Path("geo") String geo, @Query("token") String token);
}

File: app/src/main/java/com/piyushsatija/pollutionmonitor/ApiInterface.kt
package com.piyushsatija.pollutionmonitor

import retrofit2.Call
import retrofit2.http.GET
import retrofit2.http.Path
import retrofit2.http.Query

interface ApiInterface {
    @GET("feed/here/")
    fun getAQI(@Query("token") token: String?): Call<ApiResponse?>?

    @GET("feed/{geo}/")
    fun getLocationAQI(@Path("geo") geo: String?, @Query("token") token: String?): Call<ApiResponse?>?
}